tests/ui/suggestions/issue-89640.stderr

```diff
@@ -0,0 +1,13 @@
+error: expected one of `!`, `.`, `::`, `;`, `?`, `{`, `}`, or an operator, found `t`
+  --> $DIR/issue-89640.rs:2:8
+   |
+LL |     le t x: i32 = 3;
+   |        ^ expected one of 8 possible tokens
+   |
+help: did you accidentally put a space inside the keyword `let`
```

Comment:
The [suggestion style guide](https://rustc-dev-guide.rust-lang.org/diagnostics.html?highlight=illegal#suggestion-style-guide) advises against questions as suggestions. Not sure about the exact wording though, maybe:


```suggestion
help: you might have meant to write the keyword `let`
```

```suggestion
help: you might have meant to write the keyword `let` without the space
```


```suggestion
help: consider removed the space to spell keyword `let`
```